mits/automated_emails.py

```diff
@@ -6,11 +6,13 @@ def __init__(self, *args, **kwargs):
         kwargs.setdefault('sender', c.MITS_EMAIL)
         AutomatedEmail.__init__(self, MITSTeam, *args, **kwargs)
 
+AutomatedEmail.queries[MITSTeam] = lambda session: session.mits_teams()
+
 # We wait an hour before sending out this email because the most common case
 # of someone registering their team is that they'll immediately fill out the
 # entire application, so there's no reason to send them an email showing their
 # currently completion percentage when that info will probably be out of date
-# by the time they read it.  By waiting an hour, we 
+# by the time they read it.  By waiting an hour, we ensure this doesn't happen.
```

Comment:
Unrelated fix; I noticed that the end of this sentence had gotten cut off somehow.  Or possibly I just got distracted when writing it in the first place in the middle of